docs/ldoc.ltp

```diff
@@ -560,6 +560,49 @@
             target.classList.remove("open");
         }
     });
+
```

Comment:
eventually, I will use the `ldoc.include_file` for this stuff, but I haven't done enough refactoring for this yet.

Reply:
I think we can already move JS to a separate file, if that's what you mean.

If you mean splitting the actual template in multiple files, that would be great for readability!